superset-frontend/src/dashboard/components/DatasourceResultsPane/index.tsx

```diff
@@ -0,0 +1,232 @@
+/**
```

Comment:
For the naming, the ResultPane has used for the `results` tab, so we have to pick a new name for this.

Another nitpicky request is that instead of writing logic in the `index.[t|j]sx?`, we should create a new file where the index file only export modules. 

![image](https://user-images.githubusercontent.com/2016594/180472814-5724ce44-e80b-477e-a69f-51b0dc5da437.png)

Reply:
Oops, missed that part in the wiki – will change.  For the component name, I'll change it to `DrillDetailPane` like you suggested before if you think that still works.